### PR TITLE
[IMP] accounting/l10n_ec: point of sale invoicing

### PR DESCRIPTION
**Pull request by rps-odoo**
Task: [#3864339](https://www.odoo.com/web?debug=1#id=3864339&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)
Link to GDrive images folder: https://drive.google.com/drive/folders/1S5XmaPh2vnTgJAXB26PgBpnuvVgaxOL9?usp=drive_link

### CODE REVIEW

**Review by samueljlieber — APPROVED** [review]
Hi @rps-odoo, thanks again for the PR! I implemented 2/4 of your images and made a few small RST syntax edits in 5d25f4d. This PR should now be ready for review. :)

**Comment by samueljlieber** [comment]
b3de21d Rebased to resolve a conflict due to #8522 adding the **eCommerce** section to `ecuador.rst`

**Review by dade-odoo — APPROVED** [review]
I'm passing this on to the PoS master @LoredanaLrpz since she handles PoS documentation, but if you need anything from the accounting team, feel free to reach out!

**Comment by rps-odoo** [comment]
Hi @LoredanaLrpz hope you are doing good! Do you got any updates/comments on this document PR? Thanks! cc @samueljlieber

File: content/applications/finance/fiscal_localizations/ecuador.rst [hunk 1]
```diff
@@ -650,6 +650,97 @@ electronic invoice after the checkout is completed.
...
+******************************
+
+The POS cashier will be able to create a new contact for a customer who requests an invoice from the
+sale. For this process, two new fields have been added to the contact creation form. Type of
+identification and identification number.
+
+.. note::
+   A verification exists to verify that the identification number filled has the correct length. For
+   RUC identification, 13 digits are required. For Cédula, 9 digits are required. When the
+   configurations are all set, this feature will cover the next invoicing flows.
```
Comment (LoredanaLrpz) [inline content/applications/finance/fiscal_localizations/ecuador.rst:700]: A verification exists to verify sounds weird to me; you also mention a feature: do you mean a feature checks the identification number length? 
Also, I get that Cédula and RUC are mentioned as examples? It looks like a very short list. Or maybe the verification feature only covers these two?

File: content/applications/finance/fiscal_localizations/ecuador.rst [hunk 2]
```diff
@@ -650,6 +650,97 @@ electronic invoice after the checkout is completed.
...
+
+Electronic invoice: anonymous end consumer
+******************************************
+
+When clients do not request an electronic invoice for their purchase, Odoo automatically selects the
+contact for the order as the *Consumidor Final* and generates an electronic invoice.
```
Comment (LoredanaLrpz) [inline content/applications/finance/fiscal_localizations/ecuador.rst:706]:
```suggestion
When clients do not request an electronic invoice for their purchase, Odoo automatically sets the
customer as :guilabel:`Consumidor Final` and generates an electronic invoice anyway.
```

File: content/applications/finance/fiscal_localizations/ecuador.rst (hunk 1 above)
Comment (LoredanaLrpz) [reply content/applications/finance/fiscal_localizations/ecuador.rst:700]:
```suggestion
As the identification number length differs depending on the identification type, Odoo automatically checks the :guilabel:`Tax ID` field upon saving the customer form. To manually ensure the length is correct, know that the :guilabel:`RUC` and :guilabel:`Citizenship` types require 13 and 10 digits respectively.

```

File: content/applications/finance/fiscal_localizations/ecuador.rst [hunk 3]
```diff
@@ -650,6 +650,97 @@ electronic invoice after the checkout is completed.
...
+   If the client requests a credit note due to a return of this type of purchase, the credit note
+   should be made using the client's real contact information. Credit notes cannot be created to
+   *Consumidor Final*, and can be managed directly from the POS session.
+
+Electronic invoice: specific customer
+*************************************
+
+If a customer requests an invoice during the sales, it is possible to select or create the required
+contact with their fiscal information so the invoice can be created with the desired customer
+information.
```
Comment (LoredanaLrpz) [inline content/applications/finance/fiscal_localizations/ecuador.rst:718]:
```suggestion
contact with their fiscal information. This ensures the invoice is generated with accurate customer details.
```

File: content/applications/finance/fiscal_localizations/ecuador.rst [hunk 4]
```diff
@@ -650,6 +650,97 @@ electronic invoice after the checkout is completed.
...
+
+.. note::
+   If the client requests a credit note due to a return of this type of purchase, the credit note
+   should be made using the client's real contact information. Credit notes cannot be created to
+   *Consumidor Final*, and can be managed directly from the POS session.
+
+Electronic invoice: specific customer
+*************************************
+
+If a customer requests an invoice during the sales, it is possible to select or create the required
```
Comment (LoredanaLrpz) [inline content/applications/finance/fiscal_localizations/ecuador.rst:716]:
```suggestion
If a customer requests an invoice during the sales, it is possible to select or create a
```

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Hey @rps-odoo, thank you for your work, and sorry for the delay.
Here's my review and suggestions :)
Have a nice day!

**Comment by samueljlieber** [comment]
Thank you @LoredanaLrpz for your helpful review! I implemented most of your suggestions in e158e73. 

@rps-odoo can you help me with this remaining one: https://github.com/odoo/documentation/pull/8962#discussion_r1656948762

File: content/applications/finance/fiscal_localizations/ecuador.rst [hunk 5]
```diff
@@ -650,6 +650,97 @@ electronic invoice after the checkout is completed.
...
+
+Invoicing flows
+~~~~~~~~~~~~~~~
+
+Type and identification number
+******************************
+
+The POS cashier will be able to create a new contact for a customer who requests an invoice from the
+sale. For this process, two new fields have been added to the contact creation form. Type of
+identification and identification number.
```
[the remark this answers is not in the document]
Comment (rps-odoo) [reply content/applications/finance/fiscal_localizations/ecuador.rst]: Thanks for the suggestion @LoredanaLrpz . Identificación type is the field called 'Tax ID'. We can implement the following:

The POS cashier will be able to create a new contact for a customer who requests an invoice from an open session. Please refer to this documentation.

.. documentation: https://www.odoo.com/documentation/17.0/applications/sales/point_of_sale.html#set-customers

The *Ecuadorian Module for Point of Sale* adds two new fields to the contact creation form: :guilabel:`Identification Type` and :guilabel:`Tax ID`.

@samueljlieber let me know if my comment is clear :)

**Comment by samueljlieber** [comment]
Thank you @rps-odoo, I implemented [your change](https://github.com/odoo/documentation/pull/8962#discussion_r1668931106) in b679428 as well as added a seealso to link to the POS documentation on receipts and invoices in the QR code section. 

@LoredanaLrpz this PR is ready for another look, TIA 🙏

**Review by auva-odoo — APPROVED** [review]
Since @LoredanaLrpz is currently off, I reviewed her comments and made some changes accordingly. I also took the opportunity to fix a few minor issues in the eCommerce workflow section just above. I hope you don't mind.

If you’re okay with my changes, you can r+ whenever you're ready. Thanks again!

@robodoo delegate+

**Comment by samueljlieber** [comment]
@robodoo r+

**Pull request closed without merging**